phpunit/class-block-type-test.php

```diff
@@ -169,14 +174,26 @@ function test_prepare_attributes() {
 		$this->assertEquals(
 			array(
 				'correct'            => 'include',
-				'wrongType'          => null,
+				/* wrongType */
```

Comment:
What's this change about?

Reply:
It's indicating that we expect that `'wrongType'` _will not_ appear in `$prepared_attributes`.